tests/framework/common.go

```diff
@@ -18,6 +18,8 @@ import (
 	"strings"
 	"time"
 
+	"golang.org/x/exp/slices"
```

Comment:
This is the experimental package https://github.com/golang/exp

> In short, code in this subrepository is not subject to the Go 1 compatibility promise. (No subrepo is, but the promise is even more likely to be violated by go.exp than the others.)

This is probably fine for our test setup and this function in particular since its pretty simple but wanted to call it out